app/settings.py

```diff
@@ -123,7 +124,7 @@
 
 REST_FRAMEWORK = {
     'DEFAULT_PERMISSION_CLASSES': [
-        'rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly'
+        'rest_framework.permissions.IsAuthenticated',
```

Comment:
This allows anyone who is signed in to access the REST endpoints

Reply:
We may need to actually change this, since users should be able to retrieve community posts even if they're not signed in